packages/cozy-client/src/models/document/locales/index.js

```diff
@@ -0,0 +1,27 @@
+import Polyglot from 'node-polyglot'
```

Comment:
This should be added to the package.json

Reply:
Oops, good point